scripts/release.star

```diff
@@ -71,7 +71,7 @@ def get_steps(edition, ver_mode):
     steps = [
         codespell_step(),
         shellcheck_step(),
-        test_backend_step(edition=edition),
+        test_backend_step(edition=edition, tries=5),
```

Comment:
Actually the number of tries shouldn't be increased for release branch builds, but only releases. If you look at .drone.yml, you'll see that also the release branch builds change.

Reply:
@aknuds1 do we need them for `test-release`?